scripts/HealthCheck.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class HealthCheck {
```

Comment:
Can you add a comment here explaining what `HealthCheck` is for? (i.e. make it clear this is just for docker-compose and not for k8s health checks)